tests/www/test_views.py

```diff
@@ -1983,7 +1983,7 @@ def test_failed_success(self):
             past="false",
         )
         resp = self.client.post('failed', data=form)
-        self.check_content_in_response('Redirecting', resp, 302)
+        self.check_content_in_response('Redirecting', resp)
```

Comment:
This doesn't seem like the right fix. If it's saying "redirecting" but not issuing a 302 something else needs changing here.

Reply:
@ashb That test has been broken since it was created (mostly because `can_failure` was missing).

You're probably right, about 'Redirecting' what content does that POST return? 

What text should we check for, other tests in that file check for `example_bash_operator` in the response, should this do the same?